src/sentry/api/endpoints/project_user_reports.py

```diff
@@ -139,6 +140,8 @@ def post(self, request, project):
                 countdown=30,
             )
 
+        user_feedback_received.send(project=report.project, group=report.group, sender=self)
```

Comment:
This wasn't really a bug, just realized it wasn't eing triggered from this area too.